CHANGELOG.md

```diff
@@ -1,45 +1,62 @@
+## 0.2.0
+
+- Get rid of build warnings and upgrade postcss ([493293d1](https://github.com/Khan/tota11y/commit/493293d1))
+- Added relative link to license.txt ([b3da3c63](https://github.com/Khan/tota11y/commit/b3da3c63))
```

Comment:
Why do these changes link to commit instead of PRs?

Reply:
Because those changes were made outside the GitHub PR system (phabricator or otherwise).